src/main/steps/c100-rebuild/safety-concerns/abduction/child-location/content.ts

```diff
@@ -0,0 +1,93 @@
+import { TranslationFn } from '../../../../../app/controller/GetController';
+import { FormContent } from '../../../../../app/form/Form';
+import { isFieldFilledIn } from '../../../../../app/form/validation';
+import { generateContent as parentContent } from '../content';
+
+const en = () => ({
+  section: 'Safety concerns',
+  title: 'Why do you think the children may be abducted or kept outside the UK without your consent?',
+  warningText: {
+    text: 'Contact the police or social services if a child you’re responsible for is at risk of being taken out of the UK without your consent.',
+    iconFallbackText: 'Warning',
+  },
+  safetyConcernsText: 'Briefly explain your concerns, including:',
+  safetyConcernsBullet1: 'Who might take them',
+  safetyConcernsBullet2: 'Where they might be taken or kept',
+  childsCurrentLocationText: 'Where are the children now?',
+  childsCurrentLocationHint:
+    'If they’re outside England or Wales, include what country they’re in and how long they’ve been there. You don’t need to include any addresses.',
+  errors: {
+    c1A_safetyConcerns: {
+      required: 'Explain why you think the children may be abducted or kept outside of the UK without your consent',
+    },
+    c1A_childsCurrentLocation: {
+      required: 'Describe where the children are now',
+    },
+  },
+});
+
+const cy = () => ({
+  section: 'Safety concerns',
+  title: 'Why do you think the children may be abducted or kept outside the UK without your consent?',
+  warningText: {
+    text: 'Contact the police or social services if a child you’re responsible for is at risk of being taken out of the UK without your consent.',
+    iconFallbackText: 'Warning',
+  },
+  safetyConcernsText: 'Briefly explain your concerns, including:',
+  safetyConcernsBullet1: 'Who might take them',
+  safetyConcernsBullet2: 'Where they might be taken or kept',
+  childsCurrentLocationText: 'Where are the children now?',
+  childsCurrentLocationHint:
+    'If they’re outside England or Wales, include what country they’re in and how long they’ve been there. You don’t need to include any addresses.',
+  errors: {
+    c1A_safetyConcerns: {
+      required: 'Explain why you think the children may be abducted or kept outside of the UK without your consent',
+    },
+    c1A_childsCurrentLocation: {
+      required: 'Describe where the children are now',
+    },
+  },
+});
+
+const languages = {
+  en,
+  cy,
+};
+
+export const form: FormContent = {
+  fields: {
+    c1A_safetyConcerns: {
```

Comment:
please rename this attribute c1A_abductionReasonOutsideUk

Reply:
Hi @yogendraU , I think it is worth to group abduction related data inside the child of c1A_safteyConcerns which I had already done for report abuse. Please let me know your thoughts .

Reply:
@yogendraU @channa-patil @vivek-sekhar should I follow this advice ^

Reply:
Field change completed

Reply:
Yes. @ShakiraKP , lets follow the approach suggested by @vivek-sekhar , and move the abduction related data inside child of safetyconcerns

Reply:
@yogendraU All abduction screens are linear so we can just use prefix 'c1A' to group them into c1A_safetyConcerns field in userCase.
@vivek-sekhar marking this as resolved as we discussed on this.